model/repository/MonitoringRepository.php

```diff
@@ -689,14 +702,24 @@ private function prepareCondition($condition, &$parameters)
             if (in_array($key, $primaryColumns)) {
                 $whereClause .= $toLower ? " LOWER(t.$key) " : " t.$key ";
                 $whereClause .= $op;
+            } else if (in_array($this->getPlatformName(), ['mysql','sqlite'])) {
+                $whereClause .= sprintf(' JSON_EXTRACT(t.%s, \'$.%s\') %s ', self::COLUMN_EXTRA_DATA, trim($key), $op);
             } else {
-                if (in_array($this->getPlatformName(), ['mysql','sqlite'])) {
-                    $whereClause .= sprintf(' JSON_EXTRACT(t.%s, \'$.%s\') %s ', self::COLUMN_EXTRA_DATA, trim($key), $op);
+                $isLikeSearch = isset($op) && stripos(strtolower($op), 'like') !== false;
```

Comment:
I guess `strtolower` not necessary since case insensitive search `stripos` is used

Reply:
good point